http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -2054,24 +2051,33 @@ components:
           description: The labels to be placed on the runtime. Of type Map[String,String]
         userJupyterExtensionConfig:
           $ref: "#/components/schemas/UserJupyterExtensionConfig"
-        jupyterUserScriptUri:
+        userScriptUri:
           type: string
           description: >
-            Optional GCS object URI to a bash script the user wishes to run
-            inside their jupyter docker. This script runs exactly once when the cluster is first initialized. Logs from
-            this script can be found in the Leo staging bucket for the cluster. Script is run as root
+            Optional GCS or HTTP URI to a bash script the user wishes to run
+            inside their Docker container. This script runs exactly once when the runtime is first initialized. Logs from
+            this script can be found in the Leo staging bucket for the runtime. The script is run as root
             and docker --privileged.
-        jupyterStartUserScriptUri:
+        jupyterUserScriptUri:
+          type: string
+          description: DEPRECATED. Please use userScriptUri instead.
```

Comment:
maybe add the date when it's deprecated? and something like "deprecated may be removed after 6 months of deprecation"?